### DISCUSSION TITLE
Why no SPDX license identifiers in documentation files?

### DISCUSSION THREAD

**Discussion by schlotter** [zephyrproject-rtos/zephyr#60380] [General]
The Zephyr Project is a great example of making use of SPDX. Every source code file clearly states its license via the SPDX license identifier:
```
# SPDX-License-Identifier: Apache-2.0
```
(Example from [zephyr/cmake/cfb.cmake](https://github.com/zephyrproject-rtos/zephyr/blob/main/cmake/cfb.cmake))

In contrast, for documentation files this is not the case: Zephyr's ReStructured Text files do not contain a SPDX licence identifier ([Example](https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/security/security-overview.rst?plain=1)).

Only the rendered documentation contains a [footer](https://docs.zephyrproject.org/latest/services/sensing/index.html#c.sensing_sensor_ctx.sensor_ctx_ptr):
> © Copyright 2015-2023 Zephyr Project members and individual contributors. Last updated on Jul 14, 2023.

Why is this the case?

**Comment by nashif** [comment]
@kestewart is it common to have SPDX in doc files?

**Reply by schlotter** [reply]
I finally managed to talk to @kestewart on OSS EU 😀

Yes, we should have SPDX license identifiers in every file. Documentation files are under a license, too, so a SPDX license identifier should be in their header. The SPDX license identifier does not need to show up in the rendered documentation.

According to [Zephyr's Project Charter](https://www.zephyrproject.org/wp-content/uploads/2023/08/LF-Zephyr-Charter-2023.08.21.pdf), 14.c), Zephyr documentation is made available via [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).

**Reply by henrikbrixandersen** [reply]
CC: @kartben
